images/tier1/external.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="18" width="18" viewBox="0 0 18 18">
+    <g fill="#72777a" fill-rule="evenodd">
```

Comment:
Same comment here about potentially getting rid of the `<g>` element and moving the fill to the paths.